packages.json

```diff
@@ -16,6 +16,7 @@
     "maintainer": "Aaron Wolen <aaron@tiledb.com>",
     "url": "https://github.com/single-cell-data/TileDB-SOMA",
     "available": true,
+    "branch": "*release",
```

Comment:
It is the same string we sent to CZI too (in https://github.com/chanzuckerberg/chanzuckerberg.r-universe.dev/commit/6418af8aa510bb70ff4c6297f80cc91df9439270) but I can't really see any build resulting from it.  Because maybe that requires a release tag / a commit to a release branch.

Reply:
If I'm reading correctly it looks like [`remotes:::parse_repo_spec`](https://github.com/r-lib/remotes/blob/88751711e328e24ef9284109b545b5e2668f4bb1/install-github.R#L5661) just searches for the littering string `*release`.

Reply:
I actually tries a remotes::install_github() call with it and it failed but I may have picked a non-standard repo.

Reply:
Maybe we can test with a temporary dummy repo.

Reply:
I don't always bother with releases in my repo but here is one (with zero deps) where I do and it ... fails:

```sh
edd@rob:~/git/arrow-nanoarrow(main)$ dkrrr rocker/r2u:jammy        # quick container launch
root@6ea58d3deeb7:/# r -lremotes -e'install_github("RcppCore/Rcpp", ref="*release")'
Error: Failed to install 'unknown package' from GitHub:
  cannot open URL 'https://api.github.com/repos/RcppCore/Rcpp/contents/DESCRIPTION?ref=*release'
root@6ea58d3deeb7:/# 
```

Ah, but re-reading `?install_github` helps:

```sh
root@6ea58d3deeb7:/# r -lremotes -e'install_github("RcppCore/Rcpp", ref=github_release())'
Downloading GitHub repo RcppCore/Rcpp@1.0.10
Running `R CMD build`...
* checking for file ‘/tmp/remotes213a2db12a/RcppCore-Rcpp-c5b1572/DESCRIPTION’ ... OK
* preparing ‘Rcpp’:
* checking DESCRIPTION meta-information ... OK
* cleaning src
* running ‘cleanup’
* checking for LF line-endings in source and make files and shell scripts
* checking for empty or unneeded directories
* building ‘Rcpp_1.0.10.tar.gz’
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
* installing *source* package ‘Rcpp’ ...
** using staged installation
** libs
using C++ compiler: ‘g++ (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0’
g++ -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I../inst/include/      -fpic  -g -O2 -ffile-prefix-map=/build/r-base-JhpCKt/r-base-4.3.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2  -c api.cpp -o api.o
g++ -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I../inst/include/      -fpic  -g -O2 -ffile-prefix-map=/build/r-base-JhpCKt/r-base-4.3.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2  -c attributes.cpp -o attributes.o
g++ -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I../inst/include/      -fpic  -g -O2 -ffile-prefix-map=/build/r-base-JhpCKt/r-base-4.3.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2  -c barrier.cpp -o barrier.o
g++ -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I../inst/include/      -fpic  -g -O2 -ffile-prefix-map=/build/r-base-JhpCKt/r-base-4.3.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2  -c date.cpp -o date.o
g++ -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I../inst/include/      -fpic  -g -O2 -ffile-prefix-map=/build/r-base-JhpCKt/r-base-4.3.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2  -c module.cpp -o module.o
g++ -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I../inst/include/      -fpic  -g -O2 -ffile-prefix-map=/build/r-base-JhpCKt/r-base-4.3.0=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2  -c rcpp_init.cpp -o rcpp_init.o
g++ -std=gnu++17 -shared -L/usr/lib/R/lib -Wl,-Bsymbolic-functions -flto=auto -ffat-lto-objects -flto=auto -Wl,-z,relro -o Rcpp.so api.o attributes.o barrier.o date.o module.o rcpp_init.o -L/usr/lib/R/lib -lR
installing to /usr/local/lib/R/site-library/00LOCK-Rcpp/00new/Rcpp/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
** checking absolute paths in shared objects and dynamic libraries
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (Rcpp)
root@6ea58d3deeb7:/# 
```

So assuming r-universe does the same pivot we should be fine.